action.yml

```diff
@@ -13,7 +13,8 @@ inputs:
     default: 'Debug'
 runs:
   using: 'docker'
-  image: 'docker://ghcr.io/apollo-fire/ccs-build:v0.0.4'
+  image: Dockerfile
+  # image: 'docker://ghcr.io/apollo-fire/ccs-build:v0.0.4'
```

Comment:
It's commented out because it useful to be able to switch between the two methods for test purposes. One of the reasons that the PR is not intended to ever be merged back in :)